packages/sanity/src/core/form/inputs/files/ImageInput/ImageActionsMenu.tsx

```diff
@@ -37,16 +54,17 @@ export function ImageActionsMenu(props: ImageActionsMenuProps) {
 
       <MenuButton
         button={
-          <Button
-            aria-label="Open image options menu"
-            data-testid="options-menu-button"
-            icon={EllipsisVerticalIcon}
-            mode="ghost"
-          />
+          <Popover content={<Menu ref={setMenuRef}>{children}</Menu>} portal open={isMenuOpen}>
```

Comment:
I think you can remove the `MenuButton` and only use what's being passed to `button`